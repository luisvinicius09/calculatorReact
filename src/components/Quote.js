import React from 'react';
import NavBar from './Navbar';

const Quote = () => (
  <>
    <NavBar />
    <div style={{ margin: '50px', textAlign: 'center' }}>
      <h1>Hello from the real world</h1>
      <p>
        What you know you can&#39;t explain, but you feel it.
        <br />
        You&#39;ve felt it your entire life, that there&#39;s something wrong with the world.
        You don&#39;t know what it is, but it&#39;s there,
        like a splinter in your mind, driving you mad.
      </p>
    </div>
  </>
);

export default Quote;
