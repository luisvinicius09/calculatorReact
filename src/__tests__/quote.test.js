import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Quote from '../components/Quote';

it('renders the quote correctly', () => {
  const quote = renderer.create(
    <BrowserRouter>
      <Quote />
    </BrowserRouter>,
  ).toJSON();
  expect(quote).toMatchSnapshot();
});
