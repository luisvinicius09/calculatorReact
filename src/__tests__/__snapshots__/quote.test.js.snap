// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the quote correctly 1`] = `
Array [
  <div
    style={
      Object {
        "display": "flex",
        "flexDirection": "row",
        "justifyContent": "space-between",
      }
    }
  >
    <div
      style={
        Object {
          "marginLeft": "10px",
        }
      }
    >
      <h1>
        Math Magicians
      </h1>
    </div>
    <div
      style={
        Object {
          "margin": "30px",
        }
      }
    >
      <a
        href="/"
        onClick={[Function]}
        style={
          Object {
            "paddingRight": "10px",
          }
        }
      >
        Home
      </a>
      <a
        href="/calculator"
        onClick={[Function]}
        style={
          Object {
            "paddingRight": "10px",
          }
        }
      >
        Calculator
      </a>
      <a
        href="/quote"
        onClick={[Function]}
      >
        Quote of the day
      </a>
    </div>
  </div>,
  <div
    style={
      Object {
        "margin": "50px",
        "textAlign": "center",
      }
    }
  >
    <h1>
      Hello from the real world
    </h1>
    <p>
      What you know you can't explain, but you feel it.
      <br />
      You've felt it your entire life, that there's something wrong with the world. You don't know what it is, but it's there, like a splinter in your mind, driving you mad.
    </p>
  </div>,
]
`;
